scripts/build_torch_wheels.sh

```diff
@@ -76,7 +76,7 @@ function maybe_install_cuda {
 }
 
 function maybe_install_sources {
-  if [ ! -d "torch" ]; then
+  if [ ! -d "pytorch" ]; then
```

Comment:
Hmm, this is not right.
The remaining of the code seems to be assuming xla/ available (ie, bazel install), so we really need to check we are in a pytorch/ source code folder, with xla/ in it.

Reply:
Yeah I'm not entirely sure what your intent was here 😄 https://github.com/pytorch/xla/commit/a0fb73eba8f0de7f0bf35e3ed853bc221f0fc456#diff-c7a41e6d497e363c1893ac1ce097cdfbR51 but yeah we want to check we're inside `pytorch` with `xla` in it, which the above doesn't really do.

Reply:
Our structure is, when we build the source code tree, we build pytorch/ with xla/ in it.

Reply:
The intent is to verify with are already within a proper source code tree, and avoid the git pulls.

Reply:
I am well aware of that and yeah we probably want to get rid of that chunk of `maybe_install_sources` and instead ensure we're in correct dir to start installing stuff. I'll close this PR then.

Reply:
Hmm, I would not remove that code as it comes pretty handy when wanting to build from scratch.